source/clear-linux/reference/image-types.rst

```diff
@@ -35,14 +35,11 @@ Table 2 lists the currently available images that are platform specific.
    * - Image Type
      - Description
 
-   * - installer.img
-     - Preferred image of |CL| with interactive installer.
+   * - live-desktop.img or live-desktop.iso
+     - Image for live booting desktop version into memory, without requiring installaton.
 
-   * - installer.iso
-     - ISO of |CL| with interactive installer. Only for special cases where ISO image format is required (not for use with a USB key)
-
-   * - live.img
-     - Image for live booting into memory, without requiring installaton.
+   * - live-server.img or live-server.iso
+     - Image for live booting server version into memroy, without requiring installation.
```

Comment:
the live-server .iso and .img will boot to a command prompt where you will need to log into the system as root the first time.  Same comments apply from above.

Also, you need to change memroy to memory